### PR TITLE
Removes stray 'external' var that is no longer relevant breaks everything

### PR DESCRIPTION
**Pull request by jaycee**
Fixes: #2952

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7730/

**Comment by jaycee** [comment]
QA:

Ensure it runs properly on Safari, the only browser that choked on this error (though really all of them should have...)

File: jujugui/static/gui/src/test/test_app.js [hunk 1]
```diff
@@ -1345,7 +1345,6 @@ describe('App', function() {
           user: '',
           password: '',
           macaroons: null,
```
Comment (frankban) [inline jujugui/static/gui/src/test/test_app.js]: remove trailing comma?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7731/

**Comment by hatched** [comment]
Thanks @jaycee :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**